src/style/cell-styles.css

```diff
@@ -0,0 +1,184 @@
+/*  cell and cell container styles*/
+
+div.cell-container {
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-right: 5px;
+  padding-left: 5px;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  outline: #f4f4f4 1px solid;
+  width: calc(100%);
+  /* put the .cell-menu-container and .cell-row-container in a row */
+  display: flex;
+  flex-direction: row; 
+}
+
+/* These may need to be integrated into the presentation mode style sheets
+but I don't see why we would have to do this immediately.
+*/
+
+div#cells.presentation div.cell-container {
+  margin: 0 auto; 
+  outline: none;
+  padding: 0px;
+}
+
+div#cells.presentation .collapse-button {
+  display: none;
+  height: 0px
+}
+
+div#cells.presentation div.cell-row.HIDDEN {
+  display: none;
+}
+
+div#cells.presentation div.cell-menu-container {
+  display: none;
+  height: 0px;
+  width: 0px !important;
+}
+
+.cell-menu-container {
+    width: 50px;
+}
+
+.cell-type-label {
+  text-align: right;
+  color: hsl(275, 12%, 82%);
+  cursor: pointer;
+  padding: 0px 6px 0px 0px;
+}
+.cell-type-label:hover {
+  background: #fbfbfb;
+  outline: 1px solid #ddd;
+  color: hsl(275, 4%, 40%);
+}
+
+div.cell-row-container {
```

Comment:
This style can be accompanied with the following override in the presentation stylesheet:

```css
/* this resets the div.cell-row-container to not have a margin for collapse + cell label buttons,
which is an additional 50px.
*/
div#cells.presentation div.cell-row-container {
  width: 100%;
}
```

This is actually an issue right now in the presentation mode. The `div.cell-row-container` elements all have -50px in width, when I think users would expect them to be all 100% wide.

Easy fix though! Just put that declaration toward the top of the presentation mode css.